### DISCUSSION TITLE
now, is the 3.0 version stable? can i use it on production envrionment?

### DISCUSSION THREAD

**Discussion by MilkWangStudio** [apache/dolphinscheduler#11044] [Q&A] [answered]
Has anyone use 3.0 version on production envrionment?

**Comment by Tandoy** [comment, marked as answer]
It is recommended to deploy the 3.0-beta version in the development environment for testing for a period of time. As far as I know 3.0 will be released in Q3. At present, there is no plan for the community to upgrade 2.x to 3.x. If the 2.x version is deployed in the production environment and the subsequent upgrade to the 3.0 release may be more troublesome.

**Reply by MilkWangStudio** [reply]
Thanks and it's so sad o(╥﹏╥)o , maybe i can ignore 2.x and wait for 3.0-release version published. wish see it next month~
